ambry-store/src/main/java/com/github/ambry/store/BlobStoreCompactor.java

```diff
@@ -557,6 +580,21 @@ private boolean copyDataByIndexSegment(LogSegment logSegmentToCopy, IndexSegment
     return copiedAll;
   }
 
+  /**
+   * Determines if {@code copyCandidate} container in the status of DELETED_IN_PROGRESS or INACTIVE.
+   * @param copyCandidate the {@link IndexEntry} to check
+   */
+  private boolean isFromDeprecatedContainer(IndexEntry copyCandidate) {
+    IndexValue copyCandidateValue = copyCandidate.getValue();
+    boolean isFromDeprecatedContainer = false;
+    if (deprecatedContainers.contains(
+        new Pair<>(copyCandidateValue.getAccountId(), copyCandidateValue.getContainerId()))) {
+      logger.trace("{} in segment in {} is from deleted container", copyCandidate, storeId);
```

Comment:
nit: `{} in store {} is from deprecated container`.
optional: may change this to `debug` level

Reply:
Done.